### ISSUE TITLE
AdvDuplicator stuck at opening

### ISSUE THREAD

**Issue by Potatosayno** [wiremod/advdupe2#163]
This doesn't happen to every save I have, but some saves of mine which worked perfect yesterday suddenly stop working today.
I wasn't pasting or doing anything, it just started happening by random to random people on my server.
When I load them up TODAY they get stuck at opening and never reach ready "Advanced duplicator is busy" while some random props of my creation are gone, and they're random each time.
Restarting the game/server doesn't fix the issue.

![20181130161210_1](https://user-images.githubusercontent.com/44731200/49294518-d5555080-f4bb-11e8-8500-451c066a9273.jpg)

**Comment by thegrb93** [comment]
Can I have a copy of the save?

**Comment by thegrb93** [comment]
Can you describe what update version the advdupe2 was on yesterday when it was working and what it is now?

**Comment by Potatosayno** [comment]
This is the save file:
[beachslide5.txt](https://github.com/wiremod/advdupe2/files/2634240/beachslide5.txt)
I don't quite know where exactly the version is written at but this is the file I used before it happened:
[advdupe2-master.zip](https://github.com/wiremod/advdupe2/files/2634265/advdupe2-master.zip)
I have used the same version today and just today it seemed to stop working so I tried updating Wiremod & Advanced Duplicator 2 to the lastest version incase I haven't had the last version but it was the same thing.
The version I'm using now is the most updated version and it still causes that error, today morning I used yesterdays version and it still did the same.
The only difference with the updated versions that I see is "Advanced Duplicator gives out less 'advanced duplicator 2 is busy' than before but still has the same issue" which might be just my imagination.

**Comment by thegrb93** [comment]
Thanks. I'll run some tests when I get home.

**Comment by thegrb93** [comment]
Did it really work when using that .zip? That's a really recent version. I'll try it now.

**Comment by thegrb93** [comment]
Your dupe works fine for me. Can you try loading it in singleplayer?

**Comment by thegrb93** [comment]
Also can you ask the server owner to look for "advdupe2" in his server's lua_errors_server.txt

**Comment by thegrb93** [comment]
Won't be able to help until I get more information.

**Comment by Potatosayno** [comment]
Sorry I was asleep, 
Yes it was the Zip.
I think I found the problem,  I have SPC installed (Smart Prop Control) so that might be what's causing it because I just checked the lua_errors_server.txt file it seems I have this error:
[ERROR] addons/smartpropcontrol/lua/autorun/smartsauce/plugins/sv_advdupe.lua:269: this install lacks the codec version to parse the dupe (have rev 4, need rev 5)
  1. error - [C]:-1
   2. Decode - addons/smartpropcontrol/lua/autorun/smartsauce/plugins/sv_advdupe.lua:269
    3. func - addons/advdupe2-master/lua/advdupe2/sv_networking.lua:293
     4. unknown - lua/includes/extensions/net.lua:32
